src/Presentation/Factory/EventFactory.php

```diff
@@ -134,6 +135,21 @@ private function getLocationProperties(Event $event): Generator
 
         if ($event->getLocation()->hasGeographicalPosition()) {
             yield new Property('GEO', new GeoValue($event->getLocation()->getGeographicPosition()));
+            $appleLocationProperty = new Property(
+                'X-APPLE-STRUCTURED-LOCATION',
+                new TextValue(sprintf('geo:%1.6F,%1.6F',
```

Comment:
Please reuse the `\Eluceo\iCal\Presentation\Component\Property\Value\GeoValue`, but for Apple’s location property, the delimiter between latitude and longitude must be a comma instead of a semicolon.

Reply:
Added a new class. Hope this is what you wanted